src/lms-ds-loader/.pylintrc

```diff
@@ -0,0 +1,12 @@
+[MASTER]
+ignore = .venv
+
+[MESSAGES CONTROL]
+disable =
+    C0330,
+    C0326,
+    fixme,
+    line-too-long
```

Comment:
Why is line-too-long disabled?

Reply:
I copied this from Google Classroom Extractor. @bradbanister ?